Scripts~/windows/.clean-exports.ps1

```diff
@@ -1,22 +1,52 @@
 $ROOT_DIR="."
-$RUNTIME_PATH="$ROOT_DIR\Runtime"
+$STAGING_DIRECTORY="$ROOT_DIR\staging-release"
 $BUILD_ARTIFACT_PATH_PATTERN="$ROOT_DIR\com.amazonaws.gamelift-*.tgz"
+$SERVER_SDK_ARTIFACT_PATH_PATTERN="$ROOT_DIR\GameLift-CSharp-ServerSDK-*.zip"
+$RELEASE_ARTIFACT_PATH_PATTERN="$ROOT_DIR\amazon-gamelift-plugin-unity-*.zip"
 
-if (-Not (Test-Path -Path $RUNTIME_PATH))
-{
-	echo "$RUNTIME_PATH directory is not found in the working directory. Make sure you are executing the script from the project root."
-	Read-Host -Prompt "Press ENTER to continue"
-	exit 1
-}
+& "$PSScriptRoot\.verify-working-directory.ps1"
+if ($LASTEXITCODE -ne 0) { exit $LASTEXITCODE }
 
 if (Test-Path -Path $BUILD_ARTIFACT_PATH_PATTERN)
 {
-	echo "Removing build directory and tarball file(s)"
+	echo "Removing plugin tarball file(s)"
 	rm -Recurse -Force $BUILD_ARTIFACT_PATH_PATTERN
 }
 else
 {
 	echo "$BUILD_ARTIFACT_PATH_PATTERN not found. Continuing..."
 }
 
-echo "Exports clean up completed!"
+if (Test-Path -Path $SERVER_SDK_ARTIFACT_PATH_PATTERN)
+{
+	echo "Removing re-packaged server sdk zip file(s)"
+	rm -Recurse -Force $SERVER_SDK_ARTIFACT_PATH_PATTERN
+}
+else
+{
+	echo "$SERVER_SDK_ARTIFACT_PATH_PATTERN not found. Continuing..."
```

Comment:
This reads like it is an error that the pattern was not found. Maybe change it to ""$SERVER_SDK_ARTIFACT_PATH_PATTERN is already clean. Continuing...".

Also, you should use "Write-Host" instead of "echo".

Reply:
Updated error messaging to be clearer that it's not an error case. Also added `-ErrorAction Stop` to the remove actions since we want to make sure the package is cleaned during a full release script run